src/main/java/org/odpi/openmetadata/adapters/connectors/integration/eventschema/exception/UnableToCreateSchemaAttributeException.java

```diff
@@ -0,0 +1,7 @@
+package org.odpi.openmetadata.adapters.connectors.integration.eventschema.exception;
```

Comment:
Missing SPDX header

In base egeria we validate this. Unfortunately we've not done this for gradle builds, but need to - as it's a requirement we contain these headers in the code where possible (legal check)